content/en-us/tutorials/scripting/basic-scripting/intro-to-scripting.md

```diff
@@ -191,19 +191,19 @@ end
 
 In the while loop, you need to write code to wait a few seconds between the platform disappearing and reappearing.
 
-The built-in function `task.wait` can be used for this. In the parentheses the number of seconds to wait is needed: for example `task.wait(3)`.
+The built-in function `Library.task.wait()` can be used for this. In the parentheses the number of seconds to wait is needed: for example `task.wait(3)`.
 
 <Alert severity="error">
 
-Whatever you do, never make a `while true` loop without including a `task.wait` — and don't test your code before you've put one in! If you don't wait, your game will **freeze** because Studio will never have a chance to leave the loop and do anything else.
+Whatever you do, never make a `while true` loop without including a `Library.task.wait()` — and don't test your code before you've put one in! If you don't wait, your game will **freeze** because Studio will never have a chance to leave the loop and do anything else.
```

Comment:
```suggestion
Whatever you do, never make a `while true` loop without including a `task.wait()` — and don't test your code before you've put one in! If you don't wait, your game will **freeze** because Studio will never have a chance to leave the loop and do anything else.
```